docs/how-to-guides/access-monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Access Monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Prometheus](#prometheus)
+* [Alertmanager](#alertmanager)
+* [Grafana](#grafana)
+  * [Using Port Forward](#using-port-forward)
+  * [Using Ingress](#using-ingress)
+
+## Introduction
+
+This guide explains how to access the monitoring stack. Lokomotive ships monitoring stack in the form of prometheus-operator component. Learn more about the prometheus-operator component and its parameters [here](../configuration-reference/components/prometheus-operator.md).
+
+## Prerequisites
+
+- Deploy the prometheus-operator component in `monitoring` namespace. If different namespace used in the configuration then change the subsequent commands accordingly, `kubectl -n <namespace> ...`.
+
+## Prometheus
+
+Execute the command to port forward prometheus locally on port `9090`:
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-prometheus 9090:9090
+```
+
+Now open the following URL: [http://localhost:9090](http://localhost:9090).
+
+## Alertmanager
+
+Execute the command to port forward alertmanager locally on port `9093`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-alertmanager 9093:9093
+```
+
+Now open the following URL: [http://localhost:9093](http://localhost:9093).
+
+## Grafana
+
+### Using Port Forward
+
+Execute the command to port forward grafana dashboard locally on port `8080`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-grafana 8080:80
+```
+
+Obtain grafana `admin` user password by running the following command:
```

Comment:
```suggestion
Obtain the password for the `admin` Grafana user by running the following command:
```

- Grafana should be capitalized.
- Missing article.
- "Grafana admin user password" is IMO confusing as it's just a series of nouns and it's not clear what we are talking about. My suggestion above makes it clearer that we are talking about "the password which belongs to the Grafana user named `admin`".